elastic/kibana
# [ResponseOps] Investigate an error thrown when testing the ESQL rule #176518

Resolves https://github.com/elastic/kibana/issues/176518

## Summary

If an esql test query returns an `undefined` value it causes the eui grid to throw an error. This pr updates the table to handle undefined values and show `-` instead.

I also removed the actions on the columns.


### Checklist



### To verify

- Create any rule and let it run (we need to have documents that have undefined values for my example query)
- Create a new ES|QL ES Query rule 
- Write out a query that will return `undefined` values
ex. `from .kibana* | keep action.name `
- Verify that the ui does not throw an error and that you see a dash in the table
